src/pages/remote-placement.js

```diff
@@ -1,15 +1,181 @@
 import React from 'react';
 import Helmet from 'react-helmet';
+import withStyles from 'react-jss';
+import cx from 'classnames';
 
-import { Header } from '../components';
+import letsChat from '../assets/images/footer/lets_chat.png';
```

Comment:
There is another image very similar to this one which is not in the footer. I think we can have those images in a folder named "src/assets/images/messages/".